airbyte-integrations/connectors/source-stripe/unit_tests/integration/test_events.py

```diff
@@ -234,23 +235,11 @@ def _read(self, config: ConfigBuilder, expecting_exception: bool = False) -> Ent
 
 @freezegun.freeze_time(_NOW.isoformat())
 class IncrementalTest(TestCase):
-    @HttpMocker()
-    def test_given_no_initial_state_when_read_then_return_state_based_on_cursor_field(self, http_mocker: HttpMocker) -> None:
```

Comment:
This test is not valid anymore because we rely on the upper boundary of the slice, is this correct? Should we keep the test and just modify the `then` statement?

Reply:
I have rolled back this change. Removed by mistake due to a message repository issue that has already been fixed.